lib/manager/components/validation/GtfsValidationViewer.js

```diff
@@ -263,7 +265,8 @@ export default class GtfsValidationViewer extends Component<Props, State> {
             }
           </Alert>
         }
-        <Panel header={<h2>{this.messages('title')}</h2>}>
+        <Panel>
+          <Panel.Heading><Panel.Title componentClass='h3'>{this.messages('title')}</Panel.Title></Panel.Heading>
```

Comment:
ooh an h2, pretty rare
```suggestion
          <Panel.Heading><Panel.Title componentClass='h2'>{this.messages('title')}</Panel.Title></Panel.Heading>
```